api-docs/components/schemas/key.json

```diff
@@ -1,75 +1,77 @@
-"Key": {
-    "type": "object",
-    "required": ["browserSdk", "browserSdkVersion", "dateCreated", "dsn", "id", "isActive", "label", "name", "projectId", "public", "rateLimit", "secret"],
-    "properties": {
-        "browserSdk": {
-            "type": "object",
-            "properties": {
-                "choices": {
-                    "type": "array",
-                    "items": {
+{
```

Comment:
Pay no attention to most of this diff; encapsulating the schema object with `{}` is the only real change.